src/codegen/cfg.rs

```diff
@@ -156,10 +156,9 @@ pub enum Instr {
     /// Emit event
     EmitEvent {
         event_no: usize,
-        data: Vec<Expression>,
-        data_tys: Vec<Type>,
+        data: Expression,
+        size: Expression,
```

Comment:
Why is the size needed?

Reply:
If solana and substrate emit targets don't need it we can remove it

Reply:
Removing this worked just fine :tada:

Reply:
nice!